baseline-refaster-rules/src/main/java/com/palantir/baseline/refaster/AssertjIsTrue.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.refaster;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.errorprone.refaster.ImportPolicy;
+import com.google.errorprone.refaster.annotation.AfterTemplate;
+import com.google.errorprone.refaster.annotation.BeforeTemplate;
+import com.google.errorprone.refaster.annotation.UseImportPolicy;
+
+public final class AssertjIsTrue {
+
+    @BeforeTemplate
+    void before(boolean bool) {
+        assertThat(bool).isEqualTo(true);
+    }
+
+    @BeforeTemplate
+    void before2(boolean bool) {
+        assertThat(bool).isEqualTo(Boolean.TRUE);
+    }
+
+    @AfterTemplate
+    @UseImportPolicy(ImportPolicy.STATIC_IMPORT_ALWAYS)
+    void after(boolean bool) {
+        assertThat(bool).isTrue();
```

Comment:
this one is actually particularly important because loads of our other refasters assume `.isTrue()` as a starting point, but if a line of code never reaches this, then we'd never enable all the other smarter refactorings.